javaparser-core/src/main/java/com/github/javaparser/printer/lexicalpreservation/Difference.java

```diff
@@ -572,6 +572,17 @@ private void applyAddedDiffElement(Added added) {
         }
 
         if (!used) {
+            if(nodeText.numberOfElements() > originalIndex + 1 &&
+                nodeText.getTextElement(originalIndex).isComment()) {
+                String expanded = nodeText.getTextElement(originalIndex).expand();
+                if(expanded.startsWith("/*")) {
```

Comment:
mmm. I would use an helper function, maybe putting it directly on the nodetext, to check if the node text is a comment. We should also consider line comments

Reply:
It is considering line comments. You can see that both tests pass.